### DISCUSSION TITLE
How to add custom tags to the generated model files?

### DISCUSSION THREAD

**Discussion by tschwann** [99designs/gqlgen#1788] [Q&A] [answered]
I am interested in how to add custom tags to the generated models without plugins. What are my options? That is, ideally you would edit a `models_gen` model like this:
```
type Node struct {
	ID     string     `json:"id" foo:"bar"`
}
```
and `gqlgen generate` will understand to keep the foo tag as it understands how to not overwrite methods in `schema.resolvers`.

**Comment by 3nvi** [comment]
Most likely related to the proposal I opened https://github.com/99designs/gqlgen/issues/1620

**Comment by tschwann** [comment]
@3nvi, indeed that's what I am looking for!

**Comment by tprebs** [comment]
#1650 is also related to this issue. This provides a consistent way to keep tag generation as close to the schema definition as possible

**Comment by tprebs** [comment, marked as answer]
Should this issue now be closed? There are multiple ways to implement tags both using plugins (#1650 ) and without using plugins (#1173)

**Reply by wilhelmeek** [reply]
Thanks again for the great work here! Since 0.16 is released and [usage is documented on the website](https://gqlgen.com/config/#inline-config-with-directives), I'm inclined to say this particular discussion has been answered.

**Comment by dclipca** [comment]
@tprebs can it stay open till the release of 0.15? Using the master (```v0.14.1-0.20211015104003-129783590c9c```) gives me ```
generating core failed: unable to load github.com/symmetric-project/node-backend/graph/model - make sure you're using an import path to a package that exists
exit status 1```
What happens is when running ```go run github.com/99designs/gqlgen``` the graph is simply not generated with the previous graph being deleted. Then it returns this error.
